dags/tests/openshift_nightlies/util/test_manifest.py

```diff
@@ -59,7 +59,7 @@ def test_manifest_arm(self,mocked_manifest):
 
     def test_cloudreleases_amd(self,mocked_releases):
         releases = mocked_releases
-        assert len( releases ) == 145
+        assert len( releases ) == 143
```

Comment:
honestly this is a flakey assertion. You can probably remove both instead of changing.